tableauserverclient/server/endpoint/databases_endpoint.py

```diff
@@ -88,43 +169,45 @@ def _get_tables_for_database(self, database_item):
         return tables
 
     @api(version="3.5")
-    def populate_permissions(self, item):
+    def populate_permissions(self, item: DatabaseItem) -> None:
         self._permissions.populate(item)
 
     @api(version="3.5")
-    def update_permissions(self, item, rules):
+    def update_permissions(self, item: DatabaseItem, rules: list[PermissionsRule]) -> list[PermissionsRule]:
         return self._permissions.update(item, rules)
 
     @api(version="3.5")
-    def delete_permission(self, item, rules):
+    def delete_permission(self, item: DatabaseItem, rules: list[PermissionsRule]) -> None:
         self._permissions.delete(item, rules)
 
     @api(version="3.5")
-    def populate_table_default_permissions(self, item):
+    def populate_table_default_permissions(self, item: DatabaseItem):
```

Comment:
add -> None

Reply:
Actually I see there's still a couple methods needing more work, so this can be left for later.